src/main/java/de/samply/share/broker/rest/Searchbroker.java

```diff
@@ -1281,8 +1279,9 @@ public Response setSiteByName(@HeaderParam(HttpHeaders.AUTHORIZATION) String aut
    * @param queryContainer the structuredQuery and the targets
    * @return response if the query has been created
    */
+  @Secured
```

Comment:
What kind of authentication we like to use here? The monitoring (icing) authentication is not appropriate. Why not the Basic Auth with a separate structured query account?